pkg/planner/core/expression_rewriter.go

```diff
@@ -332,6 +339,9 @@ type exprRewriterPlanCtx struct {
 	insertPlan *Insert
 
 	rollExpand *logicalop.LogicalExpand
+
+	// columnVisited records the visited columns, which is used for checking columns privilege
```

Comment:
It's better to note in the comment that columnVisited only precollects all necessary columns in the request for unified permission checks at the xxx location later.